ws_fnc/AI/fn_createGarrison.sqf

```diff
@@ -204,23 +217,36 @@ for "_x" from 1 to _int do {
 	_u setPosATL _bp;
 	[_u] joinsilent _grp; //otherwise the side might be wrong
 	dostop _u;
-	//_u disableAI "PATH";
+	// _u disableAI "PATH";
 
 	_u spawn ws_fnc_setInsidePos; // SetInsidePos is fairly expensive, thus spawned
 
 	// Set new variables
 	_u setVariable ["ws_bpos",_bp,true];
 	_b setVariable ["ws_bUnits",_bu + 1,true];
-	_b setVariable ["ws_bPosLeft",_bpl,true];
+	_b setVariable ["ws_bPosLeft",_bpl,true]; 
```

Comment:
looks like trailing whitespace here.

Reply:
I don't recall modifying this, but fixed.